.github/workflows/rust.yml

```diff
@@ -27,6 +27,10 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - uses: Swatinem/rust-cache@v2
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v5
+      with:
```

Comment:
should we use latest python instead? Does it even matter? Should we even attempt to build the Python bindings, since there are separate Python workflows that test against multiple Python versions?

Reply:
no idea tbh